pkg/authorization/apis/authorization/types.go

```diff
@@ -497,3 +497,71 @@ type ServiceAccountReference struct {
 	// which the ServiceAccountReference is embedded is used.
 	Namespace string
 }
+
+// +genclient
+// +genclient:nonNamespaced
```

Comment:
don't generate this.  We're trying to scale back internal clients and this one won't work right anyway.